Firestore/core/test/unit/remote/remote_event_test.cc

```diff
@@ -577,6 +581,7 @@ TEST_F(RemoteEventTest, ExistenceFilterMismatchWithBloomFilterSuccess) {
       target_map, no_outstanding_responses_,
       DocumentKeySet{doc1.key(), doc2.key()},
       Changes(std::move(change1), std::move(change2), std::move(change3)));
+  OverrideDefaultDatabaseId(model::DatabaseId("test-project", "test-database"));
```

Comment:
Please add an inline comment above this line to document _why_ you are explicitly specifying the database ID.